src/ikea_api/endpoints/search/__init__.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import annotations
+
+from typing import Any
+from ikea_api.api import API, Method
+from ikea_api.constants import Constants
+
+
+class Search(API):
+    def __init__(self):
+        super().__init__(
+            token="",
+            endpoint=f"https://sik.search.blue.cdtapps.com/{Constants.COUNTRY_CODE}/{Constants.LANGUAGE_CODE}/search-result-page",
+        )
+
+    def __call__(self, query: str, amount: int = 24) -> list[dict[str, Any]]:
```

Comment:
Since class is quite simple and has one goal, add content of `_search` to `__call__`